### DISCUSSION TITLE
splink 3.4.2 is slower then the previous while using with spark?

### DISCUSSION THREAD

**Discussion by abhishek-ksolve** [moj-analytical-services/splink#909] [Q&A]
Hey I ran this new version of splink on a spark, And I found that the latest version of splink is much slower than the previous version 2.1.14, And also the parameters were too simple, but the newest version has complex parameters. May I know if my statement is correct or not?Is this version not made for spark?

**Comment by RobinL** [comment]
For most jobs, Splink 3 is significantly faster than Splink 2 using Spark, with less code required. Do you have an reproducible example of a job that's slower with Splink 3?
